test/test_pytools.py

```diff
@@ -664,6 +664,35 @@ def test_unique_name_gen_conflicting_ok():
     ung.add_names({"a", "b", "c"}, conflicting_ok=True)
 
 
+def test_ignoredforequalitytag():
+    from pytools.tag import IgnoredForEqualityTag, Tag, Taggable
+
+    # Need a subclass that defines the copy function in order to test.
+    class TaggableWithCopy(Taggable):
+
+        def copy(self, **kwargs):
```

Comment:
Don't you want a `_with_new_tags` here?

Reply:
Done in [9866d7d](https://github.com/inducer/pytools/pull/134/commits/9866d7d10ad7ccab057f2aa78130a8b31a06be23)